docs/settings/reporting-settings.asciidoc

```diff
@@ -104,15 +104,13 @@ security is enabled, `xpack.security.encryptionKey`.
 [cols="2*<"]
 |===
 | `xpack.reporting.queue.pollInterval`
-  | Specifies the number of milliseconds that the reporting poller waits between polling the
-  index for any pending Reporting jobs. Defaults to `3000` (3 seconds).
+  | ({ref}/common-options.html#time-units[time value]) Specifies the time that the reporting poller waits between polling the index for any
```

Comment:
We don't usually put a link like this at the beginning of a sentence. Another approach is to remove "time value" and put the link on "time" in the first sentence.   In other places, you can put the link on "how long"

Reply:
Also, use Specify instead of Specifies for consistency on the page.